README.md

```diff
@@ -29,28 +29,77 @@ Gzip running on a Pixel 6a via `adb shell`:
 
 ## Pre-requisites
 
-You will need Android NDK and since at the moment of writing
-this NDK toolchain does not support wasm target, - clang
-from Andorid OpenSource Project toolchain.
+You will need Android NDK and since at the moment of writing this NDK toolchain
+does not support wasm target, `clang` from Android OpenSource Project
+toolchain. Moreover, you need the modified `wasm2c` from the forked WABT
+repository.
+
+1. (Optional) Create a working directory to hold your copy of the NDK as well as
+   the toolchain. You can use the same directory for both, or you can use
+   different locations. (We will later tie these together using environment
+   variables. See below.)
+   
+   ```
+   cd $HOME
+   mkdir XXX
```

Comment:
maybe use something more descriptive here like `android_wasm`?

Reply:
I wanted to make it clear that the name can be freely chosen, so I wanted to avoid an "official sounding" name.  But perhaps I can change it into something "my_android_wasm"?

Reply:
sgtm

Reply:
done